x/pocket/types/codec.go

```diff
@@ -1,10 +1,11 @@
 package types
 
 import (
+	// this line is used by starport scaffolding # 1
```

Comment:
Was this you or ignite moving this?

Reply:
> Was this you or ignite moving this?

+1 and same question for similar moved ignite comments.

Reply:
It's the linter.